libr/socket/run.c

```diff
@@ -79,19 +79,22 @@ static void dyn_init(void) {
 	if (!dyn_forkpty) {
 		dyn_forkpty = r_lib_dl_sym (NULL, "forkpty");
 	}
-
-#ifdef __linux__
-	// for linux distros which use it, attempt to fall back on libutil.so if something failed
+#ifdef R_LIB_EXT
```

Comment:
This variable is always set. Better wrap this in #if __UNIX__

Reply:
thanks!